portfolio/src/App.css

```diff
@@ -14,18 +21,22 @@
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: #427aa1;
   min-height: 100vh;
```

Comment:
The header is big? Is this basically above-the-fold content?

Reply:
This I don't know. This was automatically generated with `npx create-react-app`